offline/cable_bios_met_obs_params.F90

```diff
@@ -1154,6 +1155,10 @@ SUBROUTINE cable_bios_read_met(MET, CurYear, ktau, dels )
        READ (swdown_unit) bios_rundate, swdown_day        ! Packed vector of daily AWAP/BIOS swdown (MJ)
        READ (tairmax_unit) bios_rundate, tairmax_day       ! Packed vector of daily AWAP/BIOS max air temp (deg C)
        READ (tairmin_unit) bios_rundate, tairmin_day       ! Packed vector of daily AWAP/BIOS min air temp (deg C)
+       
```

Comment:
should be removed

Reply:
Remove what?  The extra line at L1158 or the code from L1154-1156.

Reply:
I forgot about that! But it's actually not these lines I added, it's two print statements below this that is the debugging code.  I have removed it in my branch so will update in next commit.

Reply:
In case you are wondering.  I did check - these lines are in both NESP2pt9_BLAZE and TRENDY_POP branches.